Website/src/rendering/src/components/Checkout/CheckoutSummary.tsx

```diff
@@ -12,13 +14,43 @@ const CheckoutSummary = (props: CheckoutSummaryProps): JSX.Element => {
   const dispatch = useAppDispatch();
   const [loading, setLoading] = useState(false);
   const router = useRouter();
-  const { order, shipEstimateResponse, shippingAddress, payments } = useOcCurrentOrder();
+  const { order, shipEstimateResponse, shippingAddress, payments, lineItems } = useOcCurrentOrder();
   const shipEstimate = shipEstimateResponse?.ShipEstimates?.length
     ? shipEstimateResponse.ShipEstimates[0]
     : null;
   const selectedShipMethodId = shipEstimate?.SelectedShipMethodID;
 
   const onOrderSubmitSuccess = () => {
+    const orderItems: OrderItem[] = [];
+    lineItems.forEach((lineItem) => {
+      orderItems.push({
+        type: 'LINEITEM',
+        referenceId: lineItem.ID,
+        orderedAt: lineItem.DateAdded,
+        price: lineItem.UnitPrice,
+        name: lineItem.Product.Name,
+        productId: lineItem.ProductID,
+        quantity: lineItem.Quantity,
+      });
+    });
+
+    const orderCheckoutPayload: Partial<OrderCheckoutPayload> = {
+      // TODO change when possible to select language from dropdown
+      language: 'EN',
+      currency: order.Currency || 'USD',
```

Comment:
Top level CDP events language and currency are already handled by the `BoxeverService.tsx` file. From experience, Boxever does not play very well with non USD currency with our current setup. I think we could leave USD hardcoded for now and revisit that when we introduce more currencies to the demo.